Add example with the new draft and publish feature. (#6)
MattieBelt/strapi-hook-algolia: README.md

};
```

> If you are using the draft/publish feature, here is how to do it to no index unpublished content:

Reviewer: ```suggestion
If you are using the draft/publish feature, here is how to do it to not index unpublished content:
```

Typo

> If you are using the draft/publish feature, here is how to do it to no index unpublished content:

Reviewer: Or maybe this is better:
```suggestion
If you are using the draft/publish feature, here is an example on how to only index published entries:
```

Reviewer: Hey @francois2metz,

After rethinking it, I think _'If you are using the draft/publish feature, here is an example on how to only index published entries:'_ is better, could edit it into this?

Author: I did it.
